src/components/common/Header.jsx

```diff
@@ -1,10 +1,12 @@
 import Logo from "./Logo";
+import GuideButton from "@components/guide/GuideButton";
 
 export default function Header() {
   return (
     <>
-      <header className="flex items-center mx-6 py-3 border-b border-slate-200/10">
+      <header className="flex flex-rows justify-between items-center mx-6 py-3 border-b border-slate-200/10">
```

Comment:
```suggestion
      <header className="flex justify-between items-center mx-6 py-3 border-b border-slate-200/10">
```

`display: flex`설정 시 `flex-direction`의 기본값은 `row`가 적용되는 것으로 따로 설정해주신 이유가 궁금합니다!🧐